unittests/feed/FeedTestUtil.h

```diff
@@ -55,20 +59,39 @@ class FeedBaseTest : public SyncAsioContextTest, public MockBackendTest {
         MockBackendTest::TearDown();
         SyncAsioContextTest::TearDown();
     }
+};
+
+class SharedStringJsonEqMatcher {
+    std::string expected_;
+
+public:
+    using is_gtest_matcher = void;
+
+    explicit SharedStringJsonEqMatcher(std::string expected) : expected_(std::move(expected))
+    {
+    }
+
+    bool
+    MatchAndExplain(std::shared_ptr<std::string> const& arg, std::ostream* /* listener */) const
+    {
+        return json::parse(*arg) == json::parse(expected_);
+    }
 
-    std::string const&
-    receivedFeedMessage() const
+    void
+    DescribeTo(std::ostream* os) const
     {
-        auto const mockSession = dynamic_cast<MockSession*>(sessionPtr.get());
-        [&] { ASSERT_NE(mockSession, nullptr); }();
-        return mockSession->message;
+        *os << "contains json" << expected_;
```

Comment:
I assume this is going to produce a message without a space between the label and the actual content.. let's make it readable if so 👍

Reply:
Also capitalize "Contains" to match what's below 🔢